front/src/modules/timesStops/hooks/useTimeStopsColumns.ts

```diff
@@ -116,6 +116,14 @@ export const useTimeStopsColumns = <T extends TimeStopsRow>(
         disabled: ({ rowData, rowIndex }) =>
           isOutputTable || (rowIndex !== allWaypoints.length - 1 && !rowData.stopFor),
       },
+      {
+        ...keyColumn('shortSlipDistance', checkboxColumn as Partial<Column<boolean | undefined>>),
+        title: t('shortSlipDistance'),
+        ...fixedWidth(94),
+        disabled: ({ rowData, rowIndex }) =>
+          isOutputTable ||
+          (rowIndex !== allWaypoints.length - 1 && !rowData.stopFor && !rowData.onStopSignal),
```

Comment:
Should be enough as `onStopSignal` is already disabled if no `stopFor`

```suggestion
          (rowIndex !== allWaypoints.length - 1 && !rowData.onStopSignal),
```